bolt-modules/boltlib/lib/puppet/datatypes/target.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+# This was failing local test runs
+require 'bolt/target'
```

Comment:
We should fix the tests that aren't requiring this properly. The target class should come from the inventory which is injected via the puppet context. This type doesn't depend on `Bolt::Target`, which is important for use in PE.

Reply:
I don't remember which test was failing - I'll just remove this for now.